doc/api/next_api_changes/behavior/28437-CH.rst

```diff
@@ -1,8 +1,10 @@
 *alpha* parameter handling on images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When passing and array to ``imshow(..., alpha=...)``, the parameter was silently ignored
-if the image data was a RGB or RBGA image or if :rc:`interpolation_state`
-resolved to "rbga".
+When passing an array to ``imshow(..., alpha=...)``, the parameter was silently ignored
+if the image data was a RGB or RGBA image or if :rc:`interpolation_stage`
```

Comment:
This is not quite right - for RGBA the _alpha_ parameter replaced the image alpha channel.  So _alpha_ was not ignored.